public/js/themeChanger.js

```diff
@@ -10,9 +10,23 @@ function themeChanger(themeChoice) {
 	// theme object is from themeMetadata.js
 	var targetTheme = theme[themeChoice];
 	document.getElementById("themeStylesheet").href = targetTheme.css;
+	if (targetTheme.videoSource) {
+		setVideo(targetTheme)
+	} else {
+		document.getElementById("videoSource").src = "."
+	}
 	localStorage.setItem('themeKey', targetTheme.key);
 	colorButton(themeChoice);
 }
+function setVideo(themeObj) {
```

Comment:
Equivalently to my prior comment, you could also enable/disable the video stylesheet here (or, have it always loaded and add a 'video-theme' class to the div.fullscreen-bg here to control its appearance)